fe/fe-core/src/main/java/com/starrocks/lake/Utils.java

```diff
@@ -204,4 +206,21 @@ public static void publishLogVersionBatch(@NotNull List<Tablet> tablets, List<Lo
             }
         }
     }
+
+    public static long getFirstWorkerGroupByWarehouseId(WarehouseManager manager, long warehouseId) {
```

Comment:
```suggestion
    public static long selectClusterFromWarehouse(WarehouseManager manager, long warehouseId) {
```

the interface should be the final design solution oriented.

Reply:
forgot that the Cluster is already removed. so the interface may be just `getWorkerGroupIdFromWarehouse(...)`